openscap_report/cli.py

```diff
@@ -85,7 +101,7 @@ def prepare_parser():
         default="WARNING",
         choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
         help=(
-            "write debug information to the log up to the LOG_LEVEL."
+            "write debug information to the log up to the LOG_LEVEL. (default: %(default)s)"
```

Comment:
This default section causes that there is the default described twice in the man page.